realsense2_camera/launch/rs_launch.py

```diff
@@ -63,11 +63,11 @@
                            {'name': 'infra_qos',                    'default': 'SENSOR_DATA', 'description': 'QoS profile name'},    
                            {'name': 'enable_gyro',                  'default': 'false', 'description': ''},                           
                            {'name': 'enable_accel',                 'default': 'false', 'description': ''},                           
-                           {'name': 'pointcloud_texture_stream',    'default': 'RS2_STREAM_COLOR', 'description': 'texture stream for pointcloud'},                           
+                           {'name': 'pointcloud_texture_stream',    'default': 'RS2_STREAM_ANY', 'description': 'texture stream for pointcloud'},                           
```

Comment:
Why?

Reply:
This is to remove the color from the pointcloud, as we are currently not using this information but this adds 24 bits more of information to each point.